Flixter/MovieViewController.m

```diff
@@ -7,101 +7,128 @@
 
 #import "MovieViewController.h"
 #import "TableViewCell.h"
+#import "DetailsViewController.h"
 #import "UIImageView+AFNetworking.h"
 
-@interface MovieViewController () <UITableViewDataSource>
-@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@interface MovieViewController () <UITableViewDataSource, UITableViewDelegate, UISearchBarDelegate>
 
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (weak, nonatomic) IBOutlet UIActivityIndicatorView *loadingIndicator;
+@property (weak, nonatomic) IBOutlet UISearchBar *tableSearchBar;
 
 @property (nonatomic, strong) NSArray *movieData;
+@property (nonatomic, strong) NSArray *filteredData;
+
 @end
 
 @implementation MovieViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
     self.tableView.dataSource = self;
+    self.tableView.delegate = self;
     self.tableView.rowHeight = 200;
+    [self.loadingIndicator startAnimating];
+    
+    self.tableSearchBar.delegate = self;
+    self.tableSearchBar.showsCancelButton = true;
     
     UIRefreshControl *refreshControl = [[UIRefreshControl alloc] init];
     [refreshControl addTarget:self action:@selector(beginRefresh:) forControlEvents:UIControlEventValueChanged];
     [self.tableView insertSubview:refreshControl atIndex:0];
-    
-    // self.tableView.delegate = self;
+
+    [self networkRequest];
+}
+
+- (void)networkRequest {
```

Comment:
Naming is hard. 

1. I would expect something called networkRequest to return a network request object that can start / cancel / configure a network request.
2. This naming assumes that there will only ever be one network request for a controller which is unlikely. Better to call it something like, `fetchMovieData`. I like to use `fetch` for network requests and `load` for when it's reading from disk / memory but this is just my personal preference for naming conventions.